src/Bicep.Core/Semantics/Namespaces/SystemNamespaceType.cs

```diff
@@ -288,13 +288,27 @@ public static class SystemNamespaceType
                 .WithRequiredParameter("stringToFind", LanguageConstants.String, "The value to find.")
                 .Build(),
 
+            new FunctionOverloadBuilder("indexOf")
+                .WithReturnType(LanguageConstants.Int)
+                .WithGenericDescription("Returns the first position of a value within an array.")
```

Comment:
Similar to the string overload description, I think it would be worth describing how the comparison works in these two new overloads, although I'm not sure exactly how to put it. Deep comparison for arrays and objects, and exact matching for simple values?

Reply:
I followed the same description as for `contains` function. I'm happy to fix the description to make it crystal clear.